pkg/csi/manila/volumesource.go

```diff
@@ -124,3 +138,12 @@ func (volumeFromSnapshot) create(req *csi.CreateVolumeRequest, shareName string,
 
 	return share, err
 }
+
+func parseStringMapFromJson(data string) (m map[string]string, err error) {
+	if data == "" {
+		return
+	}
```

Comment:
you mentioned it must be valid `Must be a string with a valid JSON object.`
but you allow '' here .. is it on purpose?

Reply:
The "Required" column in the docs says it's not required, so it can be either empty or must be a valid JSON object. I'll try to formulate the field description better so that this behavior is reflected there. Thanks.